tests/core/filtering/test_contract_on_event_filtering.py

```diff
@@ -4,10 +4,30 @@
     is_address,
 )
 
+from web3 import Web3
+from web3.middleware import (
+    local_filter_middleware,
+)
+from web3.providers.eth_tester import (
+    EthereumTesterProvider,
+)
+
 # Ignore warning in pyethereum 1.6 - will go away with the upgrade
 pytestmark = pytest.mark.filterwarnings("ignore:implicit cast from 'char *'")
 
 
+@pytest.fixture(
```

Comment:
This looks like the same fixture from `tests/core/filtering/conftest.py`. If we remove it, do the tests fail?

Reply:
No failure when they're removed!